vcf2fhir/fhir_helper.py

```diff
@@ -597,3 +547,25 @@ def generate_final_json(self):
     def export_fhir_json(self, output_filename):
         with open(output_filename, 'w') as fp:
             json.dump(self.fhir_json, fp, indent=4)
+
+
+def _append(value_to, value_from, list_of_sub_keys, code):
+    value = OrderedDict()
+    key = None
+    is_category = False
+    if code is 0:
+        key = 'category'
+        is_category = True
+    elif code is 1:
+        key = 'code'
+    elif code is 2:
+        key = 'valueCodeableConcept'
+    for k in list_of_sub_keys:
+        if is_category:
+            value[k] = value_from[key][0]['coding'][0][k]
+        elif k in value_from[key]['coding'][0].keys():
+            value[k] = value_from[key]['coding'][0][k]
+    if is_category:
+        value_to[key][0]['coding'][0] = value
+    else:
+        value_to[key]['coding'][0] = value

```

Comment:
Just return value from here as discussed. In `generate_final_json` it would look like:
`value_to[key][0]['coding'][0] = get_category_block()`
`value_to[key]['coding'][0] = get_coding_block()`

Function name above are just for example, I don't exactly know what blocks are created here. 

Additionally, you could add unit-test for the private method created.

Reply:
ya, I tried that but then it will start to violate pep 8 rules.

foreg :
 ```
                od_value_codeable_concept_coding = OrderedDict()
                temp = i['valueCodeableConcept']['coding'][0]["system"]
                od_value_codeable_concept_coding["system"] = temp
                temp = i['valueCodeableConcept']['coding'][0]["code"]
                od_value_codeable_concept_coding["code"] = temp
                temp = i['valueCodeableConcept']['coding'][0]["display"]
                od_value_codeable_concept_coding["display"] = temp
                temp = od_value_codeable_concept_coding
                od['contained'][k]['valueCodeableConcept']['coding'][0] = temp
```
here if I replace it with a function 
```
od['contained'][k]['valueCodeableConcept']['coding'][0] = _append( i, l1, 2)
```
now it is more than 79 character (including tab/space in front)
so it would have to be done like this again 

```
temp =_append( i, l1, 2)
od['contained'][k]['valueCodeableConcept']['coding'][0] = _append( i, l1, 2)

```
so here I thought ['valueCodeableConcept']['coding'][0] in part I have in function also so why not just take it there 
so now it could be like 
```
_append(od['contained'][k], i, l1, 2)

```

Reply:
one more thing what name should I keep for in replace of _append function get_block() or get_dic() some like that please suggest some thing.

> Additionally, you could add unit-test for the private method created.

yes I will add that test case

Reply:
> one more thing what name should I keep for in replace of _append function get_block() or get_dic() some like that please suggest some thing.

Function name would depend on its purpose. Unless I don't see the body of function, I cannot suggest a good name. `...block` or `...dict` is fine by me, which `dict`/`block` is more important in function name for me.